### PR TITLE
Bravostack dev

### PR DESCRIPTION
**Pull request by bravo-stack**
Worked on the Image Upload.
refactored the Individual and the Vendor Settings Pages.
And I also refactored the modal for the removing an image.

### CODE REVIEW

**Review by michael7666 — COMMENTED** [review]
mode.value ==1? ""

pls replace with dis code

**Comment by bravo-stack** [comment]
> mode.value ==1? ""
> 
> pls replace with dis code

where should the correction be made?

File: src/user-managerment/module/individual/pages/IndividualEditProfile.js [hunk 1]
```diff
@@ -12,8 +12,10 @@ const IndividualEditProfile = () => {
   return (
     <PageWrapper>
       <div className="w-full h-full flex flex-col gap-8 py-5 overflow-auto">
```
Comment (michael7666) [inline src/user-managerment/module/individual/pages/IndividualEditProfile.js:14]: If the file is deleted remove it form d prodject

Comment (bravo-stack) [reply src/user-managerment/module/individual/pages/IndividualEditProfile.js:14]: That one wasn't deleted.

**Pull request merged**